README.md

```diff
@@ -1,2 +1,162 @@
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 Portfolio <a name="Greet Microverse"></a>
+
+# 📖 Portfolio <a name="about-project"></a>
+
+This is my project Portfolio i created it using HTML, CSS.
+
+## 🛠 Built With <a name="built-with"></a>
+
+HTML
+CSS
+flexbox
+
+
+
+### Key Features <a name="key-features"></a>
+
+ Describe between 1-3 key features of the application.
+
+Added index.html file
+Added styles.css file
+Added Linters.yml
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+ Comeing Soon
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+ Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+1- Code editor
+2- Web browser
+3- Git-scm
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+using command 
+git clone https://github.com/LizanshMathur/Portfolio.git
+
+### Install
+
+Install this project with:
+
+ command:
+
+```sh
+  cd my-project
+  npm install
+```
+
+
+### Usage
+
+To run the project, execute the following command:
+
+open index.html using live server extension.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Lizansh Mathur <a name="authors"></a>
+- GitHub: [@LizanshMathur](https://github.com/LizanshMathur)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+ Describe 1 - 3 features you will add to the project.
+
+ [ ] Add header
+ [ ] Add Footer
+ [ ] Styles
+ [ ] Add projects and contact information
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+ if you like my work just give me thumps up.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+ Give credit to everyone who inspired your codebase.
+
+I would like to thank you Microverse to give me this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+## 📝 License <a name="license"></a>
+
+No License
+
+
```

Comment:
- [ ] In the README we use the `MIT` lisence please add it and make it work whoever we click on it